wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -26,6 +25,16 @@
         </div>
         <div class="explorer__wrapper" data-explorer-menu></div>
     </aside>
+    {# Backwards-compatibility for branding_logo customisations in legacy sidebar. #}
+    {# TODO Remove in Wagtail 2.17 #}
```

Comment:
I'd add ``RemovedInWagtail217Warning`` to your comment here so Matthew can grep for this later

Reply:
Cheers! (Won't the removal happen in 2.18 now, though?)

Reply:
@gasman I was expecting the following sequence:

- v2.15 (current) – slim sidebar behind feature flag (opt-in)
- v2.16 (upcoming) – slim sidebar on by default, can opt-out to legacy sidebar
- v2.17 (upcoming + 1) – legacy sidebar removed

Does that look right?

Reply:
```suggestion
    {# TODO RemovedInWagtail217Warning Remove in Wagtail 2.17 #}
```